src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/model/ViolationTicket.java

```diff
@@ -0,0 +1,299 @@
+package ca.bc.gov.open.jag.tco.oracledataapi.model;
+
+import java.sql.Date;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/**
+ * @author 237563
+ * Represents the data contained on a BC violation ticket
+ * 
+ */
+//mark class as an Entity
+@Entity
+//defining class name as Table name
+@Table
+@Getter
+@Setter
+@NoArgsConstructor
+public class ViolationTicket {
```

Comment:
Is there a discriminator or type on here to differentiate between a scanned ViolationTicket and one that is looked up?
Or is the only way to tell is if the ocrViolationTicket json string is empty or not?

Reply:
The only way to tell is if the ocrViolationTicket json string is empty or not for now.